test/e2e/components/vl-map-action.js

```diff
@@ -1,12 +1,15 @@
 const {VlElement} = require('vl-ui-core').Test;
+const VlMap = require('./vl-map');
 
 class VlMapAction extends VlElement {
   async isActive() {
     return this.hasAttribute('active');
   }
 
   async getMap() {
-    return new VlElement(this.driver, await this.driver.executeScript('return arguments[0]._mapElement', this));
+    const map = await new VlMap(this.driver, await this.driver.executeScript('return arguments[0]._mapElement', this));
+    await this.driver.executeScript('arguments[0].ready', map);
```

Comment:
Ik denk dat dit nog altijd niet goed gaat werken, aangezien die ready promise in uw browser zit en de executeScript  niet in uw browser wordt uitgevoerd.

Reply:
ja dat had @coemans onderzocht en bleek wel zo te zijn

Reply:
Straf, dan zal executeScript misschien sowieso een await doen in de browser als er een promise wordt teruggegeven.